command/hcl2_upgrade.go

```diff
@@ -159,9 +159,6 @@ func (c *HCL2UpgradeCommand) RunContext(_ context.Context, cla *HCL2UpgradeArgs)
 	}
 
 	core := hdl.(*CoreWrapper).Core
-	if err := core.Initialize(); err != nil {
-		c.Ui.Error(fmt.Sprintf("Ignoring following initialization error: %v", err))
-	}
```

Comment:
Yeah I agree with your comment "It might also have interesting errors when the plugin is actually installed." So I went back and forth on it. I didn't read over the documentation on these errors for hcl2_upgrade but it did feel a little weird to have an error, followed by a successful file creation. 

I added some alternative text to suggest the user install missing builders before running. Since we are ignoring maybe rewording a little will make it is less weird for me. I'll revisit here.

Reply:
One thing we could do too is manually ignore the 'component not found' errors.